docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/risk_exposure/index.rst

```diff
@@ -7,3 +7,172 @@ Iron Deficiency Risk Exposure
+
+Regardless of the exposure estimation approach, the iron deficiency risk factor is defined as a **continuous distribution of hemoglobin concentrations in g/L**.
+
+.. image:: iron_risk_hierarchy.svg
+
+Global TMREL Exposure Estimation
+++++++++++++++++++++++++++++++++
+
+This estimation approach is not described in the GBD 2019 Risk Factor Methods Appendix, but is documented in the IHME stash repo for the iron deficiency risk factor, `here <https://stash.ihme.washington.edu/projects/MNCH/repos/anemia_causal_attribution/browse/iron_deficiency>`_, specifically in the `calculate_normal_hgb.py` and `nutrition_iron_exposure.py` files. For the GBD 2019 estimation of the iron deficiency risk factor exposure, the following was performed:
+
+#. Estimate the TMREL
+
+	The iron deficiency TMREL was estimated as the 95th percentile of mean population hemoglobin concentration among all GBD locations for each year-, sex-, and age-specific demographic group. If the observed mean population hemoglobin concentration for a given location was greater than the 95th percentile, then the observed population hemoglobin concentration was used as the TMREL for that location. See the specific code `here <https://stash.ihme.washington.edu/projects/MNCH/repos/anemia_causal_attribution/browse/iron_deficiency/calculate_normal_hgb.py>`_.
+
+	This value is stored as the TMREL for the iron deficiency risk factor in GBD 2019.
+
+#. Calculate iron deficiency risk exposure
+
+	The iron deficiency risk factor is calculated according to the following equation, as documented `here <https://stash.ihme.washington.edu/projects/MNCH/repos/anemia_causal_attribution/browse/iron_deficiency/nutrition_iron_exposure.py>`_ and externally validated `here <https://github.com/ihmeuw/sim_sci_maternal_anemia/blob/master/data_validation/maternal_disorders_burden/custom_paf_calculations.ipynb>`_.
+
+	:math:`exposure = TMREL - (p_* * (TMREL - mean_\text{Hgb}))`
+
+	Where, :math:`p_*` is the summed prevalence of all anemia caused by any disease except dietary iron deficiency (using the MEIDs listed below) and :math:`mean_\text{Hgb}` is the observed mean population hemoglobin value (MEID = 10487).
+
+	This exposure value is stored as the exposure for the iron deficiency risk factor in GBD 2019.
+
+	The iron deficiency risk exposure estimated in this way can be interpreted as the counterfactual population mean hemoglobin concentration in the absence of all diseases that do NOT cause iron deficiency, according to a correspondence with Nick Kassebaum, who leads the GBD team responsible for these estimates. However, it is important to note that the code used does not consider diseases that cause non-dietary iron deficiency.
```

Comment:
These two sentences are a bit hard to follow (e.g. the double negative in "absence of all diseases that do NOT cause iron deficiency") - I'm having trouble wrapping my head around which causes belong where (causing iron deficiency vs. not causing iron deficiency).

Based on the above description of "summed prevalence of all anemia caused by any disease except dietary iron deficiency," I would have expected the list of MEIDs for this sum to include the diseases from the location-specific section except for MEID 390, i.e. the diseases that cause iron deficiency (but not *dietary* iron deficiency), but it looks like those are *not* included.

I think the fact that the iron-deficiency-causing diseases are not included matches the statement that the exposure "can be interpreted as the counterfactual population mean hemoglobin concentration in the absence of all diseases that do NOT cause iron deficiency," but then it seems like p_* should be defined as "the summed prevalence of all anemia caused by any disease that do not cause iron deficiency," or something.

Also, the fact that the iron-deficiency-causing diseases are not included seems at odds with the statement "However, it is important to note that the code used does not consider diseases that cause non-dietary iron deficiency," since it seems like it's specifically excluding all these diseases.

I think I'm confused -- what am I missing???  I'm rather tired, which doesn't help...

Reply:
Yeah, I am really confused by the GBD methods used here too... what they did really doesn't make sense to me, which is why I just quoted what Nick said for the interpretation. The location-specific approach they described in the appendix is a lot more intuitive to me

Reply:
Essentially, I agree that the description of the approach and the explanation of what it represents seem contradictory

Reply:
Which is why we did not choose to follow this methodology for our previous project, we thought it was more valuable to have methods that were clear to explain and understand for the client

Reply:
OK :)